support-frontend/assets/pages/[countryGroupId]/components/thankYouComponent.tsx

```diff
@@ -201,26 +199,49 @@ export function ThankYouComponent({
 	const isRegisteredAndSignedIn = !isNotRegistered && isSignedIn;
 	const isRegisteredAndNotSignedIn = !isNotRegistered && !isSignedIn;
 
-	let benefitsChecklist;
-	if (isTier) {
-		const productDescription = showNewspaperArchiveBenefit
-			? productCatalogDescriptionNewBenefits(countryGroupId)[productKey]
-			: productCatalogDescription[productKey];
-		benefitsChecklist = [
-			...productDescription.benefits
-				.filter((benefit) => filterBenefitByRegion(benefit, countryGroupId))
-				.map((benefit) => ({
-					isChecked: true,
-					text: benefit.copy,
-				})),
-			...(productDescription.benefitsAdditional ?? [])
-				.filter((benefit) => filterBenefitByRegion(benefit, countryGroupId))
-				.map((benefit) => ({
-					isChecked: true,
-					text: benefit.copy,
-				})),
-		];
-	}
+	const getBenefits = (): BenefitsCheckListData[] => {
+		// Three Tier products get their config from the Landing Page tool
+		if (isTier) {
+			const landingPageSettings = getLandingPageVariant(
+				abParticipations,
+				getSettings().landingPageTests,
+			);
+			if (productKey === 'Contribution') {
+				return [
+					...landingPageSettings.products.Contribution.benefits.map(
+						(benefit) => ({
+							isChecked: true,
+							text: benefit.copy,
+						}),
+					),
+				];
+			} else if (productKey === 'SupporterPlus') {
+				return landingPageSettings.products.SupporterPlus.benefits.map(
+					(benefit) => ({
+						isChecked: true,
+						text: benefit.copy,
+					}),
+				);
+			} else {
+				// Also show SupporterPlus benefits
+				return [
+					...landingPageSettings.products.TierThree.benefits.map((benefit) => ({
+						isChecked: true,
+						text: benefit.copy,
+					})),
+					...landingPageSettings.products.SupporterPlus.benefits.map(
+						(benefit) => ({
+							isChecked: true,
+							text: benefit.copy,
+						}),
+					),
+				];
+			}
```

Comment:
``` suggestion
			const tierThreeAdditionalBenefits =
				productKey === 'TierThree'
					? landingPageSettings.products.SupporterPlus.benefits.map(
							(benefit) => ({
								isChecked: true,
								text: benefit.copy,
							}),
					  )
					: [];
			return [
				...landingPageSettings.products[productKey].benefits.map((benefit) => ({
					isChecked: true,
					text: benefit.copy,
				})),
				...tierThreeAdditionalBenefits,
			];
`

Reply:
Hope its ok to suggest, perhaps this could also work?

Reply:
👍 it is shorter!